fullstack-cert/js-projects/pantry-lab/user-stories.md

```diff
@@ -0,0 +1,5 @@
+1. You should implement parseShipment(rawData) that returns an array of { sku, name, qty, expires } objects.
+2. You should implement planRestock(pantry, shipment) that compares arrays and returns actions: { type: "restock" | "discard" | "donate", item }.
+3. You should implement groupByZone(actions) using reduce to bucket items into storage zones.
+4. You should implement clonePantry(pantry) that returns a deep copy so planning never mutates the original list.
+5. You should implement flagExpiring(pantry, days) that filters and sorts items expiring within the provided window.
```

Comment:
dates haven't been taught yet. so it looks like this user story will either need to be updated or removed completely. please check with Naomi for this.